mpas_analysis/default.cfg

```
@@ -1379,8 +1398,49 @@ normArgsDifference = {'vmin': -300., 'vmax': 300.}
 seasons =  ['ANN']
 
 # comparison grid(s) ('latlon', 'antarctic') on which to plot analysis
-comparisonGrids = ['latlon']
+comparisonGrids = ['latlon', 'subpolar_north_atlantic']
+
+
+[climatologyMapBSF]
+## options related to plotting horizontally remapped climatologies of
+## the barotropic streamfunction (BSF) against control model results
+## (if available)
+
+# colormap for model/observations
+colormapNameResult = cmo.curl
+# whether the colormap is indexed or continuous
+colormapTypeResult = continuous
+# color indices into colormapName for filled contours
+# the type of norm used in the colormap
+normTypeResult = symLog
+# A dictionary with keywords for the norm
+normArgsResult = {'linthresh': 30., 'linscale': 0.5, 'vmin': -100., 'vmax': 100.}
+colorbarTicksResult = [-100.,-40.,-20.,-10., 0., 10., 20., 40.,100.]
+# Adding contour lines to the figure
+contourLevelsResult = np.arange(-100., 101.0, 10.)
+contourThicknessResult = 0.5
+contourColorResult = black
+arrowsOnContourResult = True
+#ticks = [np.arange(-100., 101.0, 10.), np.arange(-100., 101.0, 10.)]
+# colormap for differences
+colormapNameDifference = cmo.balance
+# whether the colormap is indexed or continuous
+colormapTypeDifference = continuous
+# the type of norm used in the colormap
+normTypeDifference = symLog
+# A dictionary with keywords for the norm
+normArgsDifference = {'linthresh': 0.1, 'linscale': 0.5, 'vmin': -10.,
+                      'vmax': 10.}
+colorbarTicksDifference = [-10., -3., -1., -0.3, -0.1, 0., 0.1, 0.3, 1., 3.,
+                           10.]
 
+# Months or seasons to plot (Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct,
+# Nov, Dec, JFM, AMJ, JAS, OND, ANN)
+seasons =  ['ANN']
+
+# comparison grid(s) ('latlon', 'antarctic') on which to plot analysis
+comparisonGrids = ['latlon','subpolar_north_atlantic', 'antarctic', 'north_pacific']
```

I think this should just be:
```suggestion
comparisonGrids = ['latlon', 'subpolar_north_atlantic']
```